samples/sensor/adxl372/README.rst

```diff
@@ -0,0 +1,104 @@
+.. _adxl372:
+
+ADXL372: Three Axis High-g I2C/SPI Accelerometer
+################################################
+
+Description
+***********
+
+This sample application produces slightly different outputs based on the chosen
+driver configuration mode:
+
+* In **Measuring Mode with trigger support**,
+  the acceleration on all three axis is printed in m/s^2 at the sampling rate (ODR).
+* In **Polled Measuring Mode**, the instantaneous acceleration is polled every 2 seconds.
+* In **Max Peak Detect Mode**, the device returns only the over-threshold
+  peak acceleration between two consecutive sample fetches or trigger events.
+  (In most high-g applications, a single 3-axis acceleration sample at the peak
+  of an impact event contains sufficient information about the event, and the
+  full acceleration history is not required.) Instead of printing the acceleration
+  on all three axis, the sample application calculates the vector magnitude
+  (root sum squared) and displays the result in g's rather than in m/s^2,
+  together with an bar graph.
+
+
+References
+**********
+
+ - ADXL372: http://www.analog.com/adxl372
```

Comment:
Missed this on the first pass.  This renders better if there's no space before the list bullet (don't indent).

Reply:
I see - should be fixed now. 
Thanks!